dev-db/mariadb-connector-odbc/mariadb-connector-odbc-3.1.11.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS=cmake
+
+inherit cmake-multilib flag-o-matic
+
+DESCRIPTION="MariaDB Connector/ODBC"
+HOMEPAGE="https://downloads.mariadb.org/connector-odbc/"
+SRC_URI="https://downloads.mariadb.org/interstitial/connector-odbc-${PV}/${P}-ga-src.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0/3.1"
+KEYWORDS="~amd64 ~x86"
+IUSE="gnutls ssl"
+
+S="${S}-ga-src"
+
+DEPEND="=dev-db/mariadb-connector-c-$(ver_cut 1-2)*
+	dev-db/unixODBC"
+RDEPEND="${DEPEND}"
```

Comment:
Wouldn't openssl or gnutls need to be a dep for this, looking at the USE flags? Please add some sort of comment above the DEPs if not.